.github/workflows/static-checks.yaml

```diff
@@ -60,7 +60,11 @@ jobs:
           test -z "$(cat $outfile)"
 
   golangci-lint:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
```

Comment:
If cross compile works on Windows, shouldn't golanci-lint work on windows files in Linux too ?

Reply:
I checked and GOOS=windows prefix seems to work well. Specially because golanci-lint does not install on Windows with golang.